src/components/preview/preview.style.ts

```diff
@@ -1,14 +1,106 @@
-import styled from "styled-components";
+import styled, { css, keyframes } from "styled-components";
 import { margin } from "styled-system";
 import baseTheme from "../../style/themes/base";
+import { Shapes } from "./preview.component";
 
 const StyledPreview = styled.div`
   ${margin}
 `;
 
+interface StyledPreviewPlaceholderProps {
+  height?: string;
+  width?: string;
+  shape: Shapes;
+  isStatic?: boolean;
+  isLastLine: boolean;
+}
+
+const shimmer = keyframes`
+  0% { 
+    opacity: 0.1 
+  }
+  70% { 
+    opacity: 1
+  }
+  100% { 
+    opacity: 0.1 
+  }
+`;
+
+function getBorderRadius(shape: Shapes) {
+  switch (shape) {
+    case "rectangle-round":
+      return "--borderRadius400";
+    case "circle":
+      return "--borderRadiusCircle";
+    default:
+      return "--borderRadius100";
+  }
+}
+
+function getHeight(shape: Shapes) {
```

Comment:
suggestion (non-blocking): you could consolidate the returns fro the rectangle shapes here but it's not critical what you have here is more than fine

```
function getHeight(shape: Shapes) {
  if (shape.includes("rectangle")) {
    return "40px";
  }

  switch (shape) {
    case "circle":
      return "56px";
    default:
      return "14px";
  }
}